packages/lesswrong/components/review/AnnualReviewPage.tsx

```diff
@@ -190,11 +191,11 @@ export const AnnualReviewPage = ({classes}: {
               value="reviewVoting"
               className={classes.tab}
             />
-            <Tab
+            {reviewPhase !== 'NOMINATIONS' && <Tab
```

Comment:
Presumably it's fine to include UNSTARTED, RESULTS, COMPLETE

Reply:
Yeah